README.rst

```diff
@@ -18,7 +18,7 @@ Fedora system and pointing your browser at
 If you are interested in contributing to Bodhi, you can read the
 `developer documentation`_.
 
-.. _developer documentation: https://bodhi.fedoraproject.org/docs/developer/index.html
+.. _developer documentation: https://bodhi.fedoraproject.org/docs/developer_docs.html
```

Comment:
This change is unrelated, but is also actually a regression - the upcoming 3.2.0 release will make the existing link correct once it is deployed to production.